ModelBundle/Repository/RouteDocumentRepository.php

```diff
@@ -23,6 +23,16 @@ public function findOneByName($name)
     }
 
     /**
+     * @param string $redirectionId
+     *
+     * @return RouteDocumentInterface
+     */
+    public function findByRedirection($redirectionId)
+    {
+        return $this->findBy(array('name' => new \MongoRegex('/.*_' . $redirectionId . '/')));
```

Comment:
prefix and value are MongoId and can't contain "_". That's why I put it in RegEx. Not very efficient. But there should be a refacto on this indexation process as sitealias modifications are not listened to change route and redirection. I made a ticket in trello on this refacto.